extension/js/content_scripts/webmail/gmail_element_replacer.ts

```diff
@@ -437,13 +434,29 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     return $(anyInnerElement).closest('div.if, td.Bu').first();
   }
 
-  private replaceStandardReplyBox = async (editable: boolean = false, force: boolean = false) => {
+  private insertEncryptedReplyBox = async (messageContainer: JQuery<HTMLElement>) => {
```

Comment:
It seems this method does not have to be async

Reply:
Yes, thanks